src/forms/SwapForm.tsx

```diff
@@ -168,7 +183,11 @@ function SwapForm({ swap, ledgers, dispatch, disabled = false }: Props) {
             selection={alphaLedger}
             options={ledgerOptions}
             disabledOptions={[betaLedger.name]}
-            onSelectionChange={onSelectionChange("alpha_ledger")}
+            onSelectionChange={
+              onAlphaLedgerChange
```

Comment:
:wrench: Another way of doing this would be to assign a default callback when the component is created that just does nothing :)